src/Extension.php

```diff
@@ -354,7 +354,12 @@ public function payment_status_update( Payment $payment ) {
 			case Core_PaymentStatus::CANCELLED:
 			case Core_PaymentStatus::EXPIRED:
 			case Core_PaymentStatus::FAILURE:
-				$rcp_payments->update( $rcp_payment_id, $rcp_payment_data );
+				$this_pronamic_payment_id = (string) $payment->get_id();
+				$last_pronamic_payment_id = (string) \rcp_get_payment_meta( $rcp_payment_id, '_pronamic_payment_id', true );
+
+				if ( empty( $last_pronamic_payment_id ) || $this_pronamic_payment_id === $last_pronamic_payment_id ) {
```

Comment:
@rvdsteege Don't us `empty`?
https://dev.to/aleksikauppila/using-isset-and-empty-hurts-your-code-aaa
Doesn't PHPStan also advise not to use this?